packages/ui-library/src/components/thumbs-satisfaction/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useState } from 'react';
+import { ContainerBox } from '../container-box';
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+import { BackgroundPaper } from '../background-paper';
+import { colors } from '../../theme';
+import { useSnackbar } from 'notistack';
+import { FormattedMessage } from 'react-intl';
+import debounce from 'lodash/debounce';
+import { ThumbsDownEmoji, ThumbsUpEmoji } from '../../emojis';
+import { SPACING } from '../../theme/constants';
+
+export interface ThumbsSatisfactionProps {
+  id: string;
+  onClickOption: ({ id, value }: { id: string; value: boolean }) => void;
+}
+
+const StyledOption = styled(BackgroundPaper).attrs({ variant: 'outlined' })<{ $selected: boolean }>`
+  ${({ theme: { palette, spacing }, $selected }) => `
+    padding: ${spacing(3)};
+    border-radius: ${spacing(2)};
+    cursor: pointer;
+    outline-width: 1px;
+    transition: all 0.3s;
+    display: flex;
+    ${
+      $selected &&
+      `
+    background-color: ${colors[palette.mode].background.tertiary};
+    outline-color ${colors[palette.mode].border.accent};
+    `
+    }
+`}
+`;
+
+const ThumbsSatisfaction = ({ id, onClickOption }: ThumbsSatisfactionProps) => {
```

Comment:
Why not using customer-satisfaction component directly?

Reply:
It has the labels on the top, values are numbers and have to parse them. Just overall simplified streamed experience using this one, plus some styles are also differente like the text being left aligned instead of center aligned etc